recipes/canmatrix/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "canmatrix" %}
+{% set version = "0.7dev" %}
+{% set sha256 = "d06dd8849fba6417d48639454221728f6c2779f17746174438039d3e44a40073" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  
+  fn: 8ab2bc648ebeef5f9c6b7368d260cd9c1b38a7fc.zip
+  url: https://github.com/ebroecker/canmatrix/archive/8ab2bc648ebeef5f9c6b7368d260cd9c1b38a7fc.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - xlwt
+    - xlrd
+    - xlsxwriter
+    - pyyaml
+    - lxml
+    - future
+
+test:
+  imports:
+    - xlwt
+    - xlrd
+    - xlsxwriter
+    - yaml
+    - lxml
+    - future
+
+about:
+  home: https://github.com/ebroecker/canmatrix
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Converting Can (Controller Area Network) Database Formats .arxml .dbc .dbf .kcd ... '
+  description: |
+    Canmatrix implements a "Python Can Matrix Object" which describes the can-communication and the needed objects (Boardunits, Frames, Signals, Values, ...) 
+    Canmatrix also includes two Tools (canconvert and cancompare) for converting and comparing CAN databases.
+    
+    There are several importers (read) and exporters (write) for this object.
+    
+    supported file formats for import:
+    
+        .dbc candb / Vector
+        .dbf Busmaster (open source!)
+        .kcd kayak (open source!)
+        .arxml autosar system description
+        .yaml dump of the python object
+        .xls(x) excel xls-import, works with .xls-file generated by this lib
+        .sym peak pcan can description
+    
+    supported file formats for export:
+    
+        .dbc
+        .dbf
+        .kcd
+        .xls(x)
+        .json Canard (open source!)
+        .arxml (very basic implementation)
+        .yaml (dump of the python object)
+        .sym
+        .xml fibex
+
+  doc_url: 
+  dev_url: https://github.com/ebroecker/canmatrix
+
+extra:
+  recipe-maintainers:
```

Comment:
@ebroecker and @altendky -- are you ok being added as a co-maintainer of the `canmatrix` recipe on conda-forge?

https://conda-forge.org/docs/recipe.html#maintainer-role

Reply:
I'm not sure if I can help here anything. But if there are things/issues which should be solved in canmatrix - please let us know.